### DISCUSSION TITLE
关于linux服务器租用的问题。

### DISCUSSION THREAD

**Discussion by cl8600** [NonceGeek/Move-Camp#142] [Q&A] [answered]
老师好。请问linux服务器租用之后具体怎么操作呢？有没有推荐的服务器租用的地址呢？另外我在vmware中安装了ubuntu20.04，这个可以用吗？谢谢老师

**Comment by geometryolife** [comment, marked as answer]
服务器和 vmware 选择其一就可以了。如果租大陆以外的服务器，使用就方便一些。如果租国内的服务器，有时候需要挂梯子，否则有些有些依赖、包下载往往会因为网络不通畅而失败。
Linux 系统都可以用，自然 ubuntu 的也没问题。
服务器租用阿里云、腾讯云、GCP、AWS……
租服务器的操作跟使用虚拟机的操作一样，可以参考这篇文章，做一些基本配置：[使用VScode连接远程服务器进行开发](https://zhuanlan.zhihu.com/p/141205262)

**Comment by cl8600** [comment]
感谢~

Joe Chen ***@***.***> 于2022年10月23日周日 07:41写道：
